test.e2e/test-server.js

```diff
@@ -105,19 +104,8 @@ masterApp.post('/setup', function(req, res, next) {
       res.send(500, err);
       return;
     }
-
     try {
-      if (includeSchema) {
-        // the new options-based API
-        servicesScript = generator.services(lbApp, {
-          ngModuleName: name,
-          apiUrl: apiUrl,
-          includeSchema: includeSchema,
-        });
-      } else {
-        // the old API, test it to verify backwards compatibility
-        servicesScript = generator.services(lbApp, name, apiUrl);
-      }
+      servicesScript = generateService(generator, lbApp, apiUrl, opts);
```

Comment:
Please keep "the old API" branch, we need to keep it covered by unit-tests to prevent regressions.